api/authorization.go

```diff
@@ -38,9 +38,10 @@ var (
 		Transport: promhttp.InstrumentRoundTripperDuration(authRequestDuration, http.DefaultTransport),
 	}
 
-	userIdContextKey        = &contextKeys{"userId"}
-	projectIdContextKey     = &contextKeys{"projectId"}
-	isCallerAdminContextKey = &contextKeys{"isCallerAdmin"}
+	userIdContextKey           = &contextKeys{"userId"}
+	projectIdContextKey        = &contextKeys{"projectId"}
+	isProjectDefaultContextKey = &contextKeys{"isProjectDefault"}
```

Comment:
I think that we'll actually not need it, because I didn't realized that Evan migrated all data at REST, so we don't have any data without project_id in BigQuery.